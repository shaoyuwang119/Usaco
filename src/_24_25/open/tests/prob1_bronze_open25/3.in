93 91
D
DD
LLD
DDDD
DWDWD
LWWWDD
WDDWLDD
LDDWDDLD
DWLDDLWWD
DDWWWWDWDD
WDWLWWWDDWD
DWLWWWWWDWDD
WLDLWDWWLLWLD
WLLLWWLWWDLLDD
LLLDWLLLDDLDDLD
DDLDDLDLWLLWWDWD
LDWWWDDLWLWWWLWWD
WLDWWLLDLLLWWWLLDD
LDDDLDWDLDLWDWDDLDD
LDDWDDLLLLLLDLWDDWWD
WDWLWWDDDLDDWDDWDWDLD
LDLWLDDWDWWLWLLLWWDWWD
DLDWWLDLDWLWDWLLDWWDDDD
DDDDDLDLDDWLWDLWWLLLLLLD
WLDLDWDWDDWWLWDWWDLDDWLLD
LLWLLLWLLDDLWWDWLLLDWDWLDD
WDDWDDLWDLWLLLLWLDDDLWLDDDD
DLWDLWLLLDWLDDDWWLLWLWWLLWDD
LDWLWWLWDLLDDWDWDLDLWLWLLLLDD
WLLLDWDLLLDDWWLLWLLWWWDWWLLDLD
LDDDDLLDLDWDLDLWDDDLLWDLDWWWWWD
DLWWLWDWLLDLWDWLLLWLDWWWLDWWDLLD
LWLDLWLDWWLDWDDLDLLDDWDWWDWLWDLLD
LLDDWWWWDDLWDLDDLLLWDDWWDWWLDLDLDD
DWDDWLLWWDWWLDDDDDLDLLDWLDLDDDDWWWD
DDDDWWLWWLWLDDWWWDDDWDLWWWLLDWWWDWDD
LLLWDDLDLDLWWDWWWWLWLWDLDLWWDDWWWDDDD
DDLLLLDWLWWLDDWLDWWDDDDDDWWLLDDWLLLLWD
WWWDDWDLDWWWWLLLLLWDWWWLWDDDLDWDLLDLDDD
DWDDDLLDLLLWDDWWWLDWWLLLDWDDWDWDDWWDWDDD
WDDDDLLDLDLWWLWDWWDLWDWDWLLWLDDDWWLLLDWWD
DDWLWLLWDDDLWWWDDDWWLDLDLDLWDLWWWLLDWWLLDD
WWLDLLDDWLDDLWLWLDWWLDLDDWWWDWWWDDLWWLDLLDD
DLWDLLDLLDLLWWWLWLWLDWLDDWDWLLLLLWLLLWDWDDDD
WLLLDDWWLLWWWLDLDLWLLWWWLLWLWLDLLWLLDDLWLLWWD
WWDDLDLLWWWLWDDWLLLLWLLDLDWWDDDDWWDWDLWWLDLWLD
DDWWWLLDLDDLDWDDDDDWWLWLLLDLLWWDLDDLDDLLDLDDLLD
WWWLWWWWDWDLWWDLDLWWWWLLWLLDLLLLWDDDLDLWLWWDWWDD
DWWWDWDDWLLDDDWDWLWDDWWDDWLLDLLLWWLLWDLLLWWWLLLDD
DWWLDDWLLWDWLDLLDWDLWDWDWWDWDDLLLWDWLLWLWDWWWLWDLD
LWDWDLWWDLWDWWDDDLWLLLWLLWLDDLWLDDWWWWLWWDDLDWDWLWD
WLWWLDWLWWDLWDLWLWLDDDWDLDDDLWLDDWLLDLDDDDDDDDLLDLLD
LLDWWDDDDLDDDWDDLWLWLWWDWLDWLLWDDWDDDDWLWLWDLLWWLLWWD
DWLWWLLLDDLDLDDLLDWWWWLWLWWLLDDDWDWDWDWWLLDWDLDWLLDWLD
DWWLWDDDDDDWDWDWDLDWDDDWLDWLLLWLDDDWWWWWDLLDDLWDWDDWLDD
DDLLDLDDWWLLDLLLWLWLDLDDDWLWDDLWWLDWDWDWDLWDDDDWLWLWDDWD
DWLDWDWLWDWLLLLDLWDDLLDLLWWWWWLDLLLLWWDWDLWWLWLWWDLDWLDLD
DDLWWLWWWDDDDDWDDDDLLWLWLWLLWDWWWWWLDLDLDDDWWDLDWWWDWDDWWD
DDDLLLDDWLDDLLDWLWWDLLWWDDDWWLWDLDDWWWDDDLWDWWWDWDWDDWWLLLD
WWDWWDLLDLLLWLWDLDDLDDLWLWLDLWWLWWDWWDLLLLWWWLWDDLDWDDDDLDLD
WLLLLLLWDDDWDWWLWWWDLLDLLDWDDDWDLWDLWWWWDWDDWDLLWLWDWDLDLDDWD
DWLLWDLWWWLDLDLLDLWWWWDLDLWDWWLDDLWWWDDWLDDDLLDWWDLWDWLLWLLLWD
DLLLWWWLDDDLDLDDDDWDDWLDLWLDDLLLLLDLDWWWLWWDDWLLLDDWWWDDLDWLWDD
WWLWLDWWWDLLWWDDDDDDLLLWLWLDWDDDLDDLLWWDLWWWDDWLDLDWLLWLDDDWWDLD
WDLLWLWLWDLDWLDWDWWWDWDWWDDWWLDWLLLDDLWLWWLDDWLDLDDLWLLLWLDDWWLLD
LDDDDWDDWLLWLWWWLDDLWDWWLLLWLWLWWWLWLWDWLDWLWLWWWLLLLDLWDLWDWWDDWD
WDWDLLWWDLDWWWDLDWLDDLDLLWLDDWDWDDWWDLDLWWWDDLDLLLLDWLWDDLLWLLWLDWD
WLLDLWLDWDLWLWDDLLLLWLWDLDWWDWDLLDWDWWLDWDLDLDDLWLWLWDWWLLWDWDWLLLLD
DDLWDWWDDDLWWLWWWDDDDLWDWDDLWDLWWLDDWDWWLWDWWWDDDWWLDLWLLDWLLLLLWLDDD
DLDDWWDWDLLWLWWLWWDDDDLLLDDDDLWWLWDDDLLLDLWLLWWDWDDWWWLWWLWLDDWDWWLWWD
LLWLLDDLLLLWDWLWLWDLWWLDWLLLDDDWWDLWLLWDLLWLWDDLLDLLWDLDWWDDDLWDWWDWLDD
LDDLDLLDWWWWWDDWDWWLWDDWWLLDLDDWLWLDWWLDWWLLLDLLWLWLWLDLLWWDDLWDDLWDDDLD
WDDLDWLLWDLLLLDLWLWWDWDLWWDWWDLDDDDDWDWDLDWLDWWWWWDWWLDLLDLWWDDWLLWLWLLDD
DWDLDDLDWLWDWWLLWWLWLWWWDWWWLDLDWWLLWLDWDWDLWLDWLWWLWDLDLWDWWLDLDWLDDWDDWD
WWLWDWWLLDLWWWLLWLDDLWWDLDLWWDLLDLLDLDWDLLLWWDWWDDWWDDDWDLWDWWDWLWDLLDDLLWD
LDDLWWWLDWLLDLWWDWDLWDLDDDDDWLLWLDWWWWWDDLDWWDWDLDWWWLWWDWDWWLDWWDWDWDDDWLLD
DWDWLLLDLWLDLWDDLDLWWDLLDDDDDWDWWWLDWLWWWDWLLWLLLWWLDLDDDDWDLWLLWLWLLDLWLLDWD
WDDDWLWDDWLWWLLWDWLLDLWLDDDWDDDLLLLDWDLDWWWDWDLWLWWDLWWLLLDWWDWWDLDDWWLWWDDWLD
WWLWLDWWWDWLWDWWLDLDDDLWWDWDLWDLDDLDDDDDLWDLLWLDDDLLDWWWLLDWDLLDWDLWLDLWLWWLWWD
WLWWLWWLDLLWLLDDDLWWLWWDDWWWDLDLDLLDDWDWWLDWDDDWDWWDDLLWLWLLWLLWWDLWDDWWDDLLDWDD
DLLWLDWWLLLWLWLLLDWDLLDWDWWWDWWWDDWLLWWDWWWLLDDDWLDWDLWDLWDWWWWDLLWLWWWDLLDLWDLLD
LDDDLLDDLLLLLWWWWLWWDLDWWLDDDDWWWLDDLWWWLWLDDWLWWDWLWDDWLDWWWLLDWWDWLWLWWLLLDDDWDD
LLWWLWDLWWLDLDLLDLDDWWWDLLWLLWLDDWDLWDLDLDWWDLWDDDLWWLWLLLDWDWDDWLWWLWLLWLWWLDLLLLD
LLLLLLWLWLLLLDLWDDWWDDWLWDWWLWWDLLDLDDWLDLDLLDWDDDDDLLDDLLWDWWWDWDWLDLDDLLDLWLDLDWDD
DLDLWDWWWDDLWWWLDDDDDDDLWDLLLWDDDDLLLLWDWWDWDDLWLLLLLWDWLDLLDDDDDWWWDLLLWLWLWDWLDLWWD
WDDLLDLDDWWDWLWLLDWDWDDWLWLDDWDLWWLLDLDDWWLDWWWLWLWWWLLWDDDWDWWWLLDDWLDDDLDLDDWDWDLLWD
LLWDDLLWDWDDWLLDDLLDDWLLLLWWLDLDDLLLDWLLDDWLLDWDLDWLDWWDLDLDDLWLWDLLDWWLWWWWLWWDWWDDDDD
DDLWLWDLDWLWLDLDLWLWLWDLLWWWWWLWLLWLLWWLWDLWLDDDWDLWWLDDWLWLLLLLDLDWLLWWWDLDDWLWDDWLLLLD
WDWWWDLWDLLLWDWLLLLLWLDLDDWLLWWDDLWLLLLLLDLWLLWLDDDWLDLLWDLDDDWWLWWLDLWLLDDLLLWLLWDWLDLLD
DWLWWWDLWLWDDWLLLWLLLWDLWWWLLWLDWDDLDDDLLLDDLDLDWLDWDLWDDLLWLWWWLWDWDLLWWDWDLWLDLDWDDLWDDD
DDWDDDLLDLDWDWDDWDDDDDWLLDDLDDDDWLWDWLWDLDDWDDWWLLDDDDWLDWDWWWLDWWLDWDWDWLLWDWWLDWLLWDWWDWD
LLDWLDLLLWDWWLWWLDDLWLDLLDDWDLLLLDWWWDLLLDDWDDDDDDLDLWLLDDLDWLWLWWLWWLWDWLLLLWDDLWWDDWLLWWDD
LWWWLDDLLWWDDWLWWWLWWLLDDWWLWDWWWDLLLLLDDLWWDDLWDLWLDWLDDLWWLDDDDWDLDDLWDWDLWLDDDLWLLLDLDLLDD
8 23
75 49
28 46
83 55
10 36
53 40
71 52
32 24
88 7
13 48
92 46
33 86
12 42
1 69
43 21
47 11
38 73
49 64
7 13
81 36
79 45
86 49
67 57
20 6
3 46
74 12
87 3
35 74
19 40
5 82
26 13
90 67
88 22
60 6
74 85
30 78
55 79
17 77
61 82
73 16
64 92
67 43
87 17
55 59
32 11
11 20
79 83
8 60
86 77
12 26
35 86
24 45
1 36
15 54
40 69
28 81
1 31
73 35
22 87
32 50
1 40
67 6
21 68
74 73
86 52
15 30
41 62
15 91
48 28
79 23
46 93
13 28
42 63
45 55
15 39
21 65
86 80
41 77
62 54
10 30
92 82
84 21
63 28
74 26
45 5
22 21
89 19
12 72
73 30
42 18
60 49
