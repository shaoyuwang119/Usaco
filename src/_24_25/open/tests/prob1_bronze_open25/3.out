905
2088
2249
1760
1424
1424
2408
2088
2565
2720
2088
1593
2088
1760
1760
1593
2249
1253
2565
728
1593
1253
1424
2088
368
1080
1925
905
2408
2088
2408
2088
2408
2088
1593
1593
1253
1424
2249
2088
1593
2408
1593
1080
1424
728
1593
2408
1760
1760
1080
2408
1080
2408
1760
2565
1424
1424
1925
1760
1593
1253
1593
1253
2249
1925
1424
905
2720
1593
1424
3024
2408
1925
1925
1925
1253
1424
1424
1080
1760
1593
2408
1925
2088
1253
2565
1593
2088
1760
2720
